.github/workflows/pre_merge.yml

```diff
@@ -21,18 +21,19 @@ jobs:
     runs-on: [self-hosted, linux, x64]
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
What's the meaning of `v3`?

Reply:
![image](https://user-images.githubusercontent.com/5232387/211718863-452e9dc3-e52f-497b-aab9-3a868e09d80a.png)
to resolve these warning message for the checkout actions.